### PR TITLE
feat: http-sparql-out

### PR DESCRIPTION
**Pull request by jobulcke**

### CODE REVIEW

File: ldi-nifi/ldi-nifi-processors/http-sparql-out-processor/pom.xml [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+        </dependency>
+        <dependency>
+            <groupId>be.vlaanderen.informatievlaanderen.ldes.ldi</groupId>
+            <artifactId>ldi-nifi-common</artifactId>
+            <version>${project.version}</version>
+        </dependency>
+
+        <dependency>
+            <groupId>com.github.tomakehurst</groupId>
+            <artifactId>wiremock-jre8</artifactId>
```
Comment (Yalz) [inline ldi-nifi/ldi-nifi-processors/http-sparql-out-processor/pom.xml:29]: test scoping

File: ldi-orchestrator/ldio-application/pom.xml [hunk 2]
```diff
@@ -380,11 +380,11 @@
...
-            <id>ldio-virtuoso-out</id>
+            <id>ldio-http-sparql-out</id>
             <dependencies>
                 <dependency>
                     <groupId>be.vlaanderen.informatievlaanderen.ldes.ldio</groupId>
-                    <artifactId>ldio-virtuoso-out</artifactId>
+                    <artifactId>ldio-http-sparql-out</artifactId>
                     <version>${project.version}</version>
                 </dependency>
             </dependencies>
```
Comment (Yalz) [inline ldi-orchestrator/ldio-application/pom.xml:390]: also skolem transformer

**Review by Yalz — APPROVED** [review]

File: ldi-nifi/ldi-nifi-processors/http-sparql-out-processor/pom.xml (hunk 1 above)
Comment (jobulcke) [reply ldi-nifi/ldi-nifi-processors/http-sparql-out-processor/pom.xml:29]: scoped by parent

File: .github/ldio.Dockerfile [hunk 3]
```diff
@@ -32,7 +32,7 @@ COPY ./ldi-orchestrator/ldio-connectors/ldio-console-out/target/ldio-console-out
 COPY ./ldi-orchestrator/ldio-connectors/ldio-http-out/target/ldio-http-out-jar-with-dependencies.jar ./lib/
 COPY ./ldi-orchestrator/ldio-connectors/ldio-noop-out/target/ldio-noop-out-jar-with-dependencies.jar ./lib/
 COPY ./ldi-orchestrator/ldio-connectors/ldio-repository-sink/target/ldio-repository-sink-jar-with-dependencies.jar ./lib/
-COPY ./ldi-orchestrator/ldio-connectors/ldio-virtuoso-out/target/ldio-virtuoso-out-jar-with-dependencies.jar ./lib/
+COPY ./ldi-orchestrator/ldio-connectors/ldio-http-sparql-out/target/ldio-http-sparql-out-jar-with-dependencies.jar ./lib/
```
[the remark this answers is not in the document]
Comment (jobulcke) [reply .github/ldio.Dockerfile:35]: 🙈

File: ldi-orchestrator/ldio-application/pom.xml (hunk 2 above)
Comment (jobulcke) [reply ldi-orchestrator/ldio-application/pom.xml:390]: 🙈

**Pull request merged**